src/math/curve-iterator.js

```diff
@@ -0,0 +1,225 @@
+Object.assign(pc, (function () {
+    'use strict';
+
+    /**
+     *
+     * @constructor
+     * @name pc.CurveIterator
+     * @classdesc CurveIterator performs fast evaluation of a curve by caching
+     * knot information where possible.
+     * @description Creates a new curve iterator.
+     * @param {Curve} curve The curve to iterator over
```

Comment:
I think the type of the parameter should be pc.Curve not just Curve

Reply:
docs have been removed.